lib/support/windows.js

```diff
@@ -92,13 +92,13 @@ const WindowsSupport = {
   },
 
   arch() {
-    if (this.cachedArch) { return this.cachedArch; }
-    try {
-      this.cachedArch = String(child_process.execSync(ARCH_BAT)).trim();
-      return this.cachedArch;
-    } catch (err) {
-      return '';
-    }
+    if (!this.cachedArch) { this.cachedArch = os.arch(); }
+    
+    // for backwards compatibility, we don't just return the direct result of the os call
+    if (this.cachedArch === 'x86') { return '32bit'; }
```

Comment:
Should this be `this.cachedArch === x32`? See https://nodejs.org/api/os.html#os_os_arch for possible values.